homeassistant/components/axis/device.py

```diff
@@ -65,80 +65,80 @@ def __init__(
         self.additional_diagnostics: dict[str, Any] = {}
 
     @property
-    def host(self):
+    def host(self) -> str:
         """Return the host address of this device."""
-        return self.config_entry.data[CONF_HOST]
+        return self.config_entry.data[CONF_HOST]  # type: ignore[no-any-return]
```

Comment:
```suggestion
        host: str = self.config_entry.data[CONF_HOST]
        return host
```

Do something like this instead of ignoring the no-any-return, . I much prefer to avoid ignores :)

Reply:
Done! My default is usually the `type: ignore` just in case the typing improves later one, this will make it more obvious as mypy will highlight usused ignores.

_In this case it would be if `config_entry` ever becomes generic so `data` can be typed as `TypedDict`._

Your suggestion does also work of course.